### PR TITLE
Add test for reports

### PR DESCRIPTION
**Pull request by lehinevych**
{{pytest: cfme/tests/test_rest.py -k 'report' -v --long-running -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a436165f9213d9ffc097c3ddc8e766f83d602ec4:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 153631470624a21344a9bb2cbbf39779a5636731:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c2795aa47f0495f6bba2130424c9fb4ed7b83586:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/test_rest.py [hunk 1]
```diff
@@ -1129,6 +1129,48 @@ def test_delete_tenants(rest_api, tenants, multiple):
             tenant.action.delete()
 
 
+@pytest.mark.uncollectif(lambda: version.current_version() < '5.5')
+def test_run_report(rest_api):
+    report = rest_api.collections.reports.find_by(name='VM Disk Usage')[0]
+    response = report.action.run()
+
+    def _finished():
```
Comment (mfalesni) [inline cfme/tests/test_rest.py]: You can do a nice handy wait_for wrapper here :smiley: 
```python
@pytest.wait_for(timeout="5m", delay=5, message="REST running report finishes")
def _finished():
    # ...
```

**Comment by dajoRH** [comment]
Lint report for commit 5c2e0656ce7c990f9a74d809f669911228f2d041:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**